wgsl/index.bs

```diff
@@ -5274,6 +5277,123 @@ A phony-assignment is useful for:
   </xmp>
 </div>
 
+### Complex assignment ### {#complex-assignment-sec}
```

Comment:
Bikeshedding (feel free to ignore): I prefer "compound assignment" to "complex assignment". The former seems to be much more commonly used when describing this feature in other programming languages, and just feels more fitting IMO.

Reply:
Me too!
Also "Complex" can be confused with complex numbers.